identity-service/src/notifications/sendNotificationEmails.js

```diff
@@ -60,6 +60,10 @@ async function processEmailNotifications (expressApp, audiusLibs) {
     for (var announcement of appAnnouncements) {
       let announcementDate = moment(announcement['datePublished'])
       let timeSinceAnnouncement = moment.duration(currentTime.diff(announcementDate)).asHours()
+      // If the announcement is too old skip it, it's not necessary to process.
```

Comment:
could we do this filtering on line 53? like literally remove it from the appAnnouncements array and all processing that happens after this? cause the situation that could arise is the data is still in the array, but it's skipped and someone could use it access it later by accident

Reply:
this is a good point, but i don't understand how the data could be accessed by someone later? the suggestion makes sense as a cleaner option but functionality seems identical to me

Reply:
@hareeshnagaraj it is functionally identical, but i wsa thinking it was a safety feature. like here https://github.com/AudiusProject/audius-protocol/pull/713/files#diff-7789276e9b185b5cd370bcebc28ccf73R85 what if the older than 10 days filter wasn't there

Reply:
Yeah it shouldn't make a difference, but we can't modify the array if that's what you're suggesting, only run a pure function to filter out the old items. 
Because the the notifications are used other places such getting it when the user scrolls down the notification panel on the dapp @dmanjunath you still think we need to modify it?